### PR TITLE
feat: add label field on mobile menu

### LINKED ISSUE #450: Add the labels on the mobile menu
### Describe the feature in detail (code, mocks, or screenshots encouraged)

I want to add the label next to the links in the mobile menu like on the bigger screen (`aside` section). It should look like this on mobile

image

### What type of pull request would this be?

Docs

### Provide relevant links or additional information.

_No response_

### PR DESCRIPTION
**Pull request by NDOY3M4N**
This closes #450

### CODE REVIEW

File: apps/www/src/lib/components/docs/nav/mobile-nav.svelte [hunk 1]
```diff
@@ -49,6 +49,13 @@
 										class="text-muted-foreground"
 									>
 										{item.title}
+										{#if item.label}
+											<span
+												class="ml-2 rounded-md bg-[#adfa1d] px-1.5 py-0.5 text-xs leading-none text-[#000000] no-underline group-hover:no-underline"
```
Comment (leonardsimonse) [inline apps/www/src/lib/components/docs/nav/mobile-nav.svelte:54]: Isn't `group-hover:no-underline` redundant as `no-underline` already has the same effect?

**Review by leonardsimonse — CHANGES_REQUESTED** [review]

File: apps/www/src/lib/components/docs/nav/mobile-nav.svelte (hunk 1 above)
Comment (NDOY3M4N) [reply apps/www/src/lib/components/docs/nav/mobile-nav.svelte:54]: Oh you're right. I just copied the markup from `docs-sidebar-nav-items.svelte`. I'll remove these 2 classes here.

Comment (huntabyte) [reply apps/www/src/lib/components/docs/nav/mobile-nav.svelte]: Whoops!

**Review by huntabyte — APPROVED** [review]
Thank you!

**Pull request merged**